config.go

```diff
@@ -112,10 +119,6 @@ func (m *Defender) Validate() error {
 	if m.responder == nil {
 		return fmt.Errorf("responder not configured")
 	}
-	if len(m.Ranges) == 0 {
```

Comment:
If I understand correctly, the new behavior would be to use all ranges, right?

This should be documented in the README.

Reply:
Always been the behavior you can look at the provision step to see where it's set.

Reply:
I'll make a PR to document that behavior.